package.json

```diff
@@ -22,13 +25,12 @@
     "dist/**/*.d.ts"
   ],
   "devDependencies": {
-    "@babel/cli": "^7.14.8",
     "@babel/compat-data": "^7.15.0",
-    "@babel/core": "^7.15.0",
+    "@babel/core": "^7.16.12",
```

Comment:
Presumably that bump means tha these need to be bumped to 7.17 or whatever

Reply:
done